src/test/java/org/tarantool/ClientReconnectClusterIT.java

```diff
@@ -0,0 +1,154 @@
+package org.tarantool;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.function.Executable;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.tarantool.AbstractTarantoolConnectorIT.makeClientConfig;
+import static org.tarantool.TestUtils.makeInstanceEnv;
+
+public class ClientReconnectClusterIT {
+    private static final int TIMEOUT = 500;
+    private static final String LUA_FILE = "jdk-testing.lua";
+    private static final String SRV1 = "replica1";
+    private static final String SRV2 = "replica2";
+    private static final String SRV3 = "replica3";
+    private static final int[] PORTS = {3302, 3303, 3304};
+    private static final int[] CONSOLE_PORTS = {3312, 3313, 3314};
+    private static TarantoolControl control;
+
+    private static String REPLICATION_CONFIG = TestUtils.makeReplicationString(
+        AbstractTarantoolConnectorIT.username,
+        AbstractTarantoolConnectorIT.password,
+        "localhost:" + PORTS[0],
+        "localhost:" + PORTS[1],
+        "localhost:" + PORTS[2]);
+
+    @BeforeAll
+    public static void setupEnv() {
+        control = new TarantoolControl();
+        int idx = 0;
+        for (String name: Arrays.asList(SRV1, SRV2, SRV3)) {
+            control.createInstance(name, LUA_FILE,
+                makeInstanceEnv(PORTS[idx], CONSOLE_PORTS[idx], REPLICATION_CONFIG));
+            idx++;
+        }
+    }
+
+    @AfterAll
+    public static void tearDownEnv() {
+        for (String name : Arrays.asList(SRV1, SRV2, SRV3)) {
+            control.stop(name);
+            control.deleteInstance(name);
```

Comment:
I think we should leave files after the test run, because it can be useful to investigate a fail. But TarantoolControl should provide such function to allow a user (a test case writer) to drop an instance entirely and run another one with the same name. Proposed fix in fe36d00c5cdfa3281f3481362d5561e6270d4bd7.

Reply:
Now removed.